lib/elixir/lib/access.ex

```diff
@@ -305,6 +305,32 @@ defmodule Access do
       "could not put/update key #{inspect key} on a nil value"
   end
 
+  @doc """
+  Removes the entry with a given key from a struct, map, or keyword list.
```

Comment:
I'm not sure it should say "struct" here, since a struct has static keys and thus any `pop` operation would actually remove a key from some data contained in the struct, rather than from the struct itself. Perhaps it would be better to just say "container"?